### ISSUE TITLE
What is the future for this plugin?

### ISSUE THREAD

**Issue by mezzaninegold** [studiopress/genesis-custom-blocks#182]
I can see a number of messages now saying that you are no longer accepting new feature requests, does this mean the plugin has stopped being developed?

Should we stop using it and look for an alternative?

We are paying to Genesis pro just for the custom blocks as we have it on multiple sites.

There are some missing fields and the layouts no longer work in the backend.

Thanks,

**Comment by mikemanger** [comment]
I believe a lot of this plugin's functionality is replaced by Advanced Custom Fields PRO (specifically [ACF Blocks](https://www.advancedcustomfields.com/resources/blocks/)), which WP Engine bought a while back.

Having some official timeline or statement about the Genesis Framework would help a lot.

**Comment by via-lars** [comment]
Hello,

I like to inquire about the future too. Can we expect issues arising from WP updates such as https://github.com/studiopress/genesis-custom-blocks/issues/184 to be handled?

We use Genesis Custom Blocks Pro for many clients and like it very much.

GCB and ACF both belonging to WP Engine I can understand only 1 "custom block creator" will be supported in the future, is there some outlook for ACF getting feature parity with GCB ?

**Comment by mezzaninegold** [comment]
Agreed, keen to get a response on this so we can plan for future projects.
